package.json

```diff
@@ -32,6 +32,7 @@
     "ssb-no-auth": "^1.0.0",
     "ssb-ref": "^2.14.3",
     "ssb-room": "^1.3.0",
+    "ssb-social-index": "^1.0.0",
```

Comment:
This needs to reference the open PR like [this](https://github.com/arj03/ssb-browser-core/blob/ce00db41e69f1d2f36046987cdcaa3294521adaf/package.json#L27) until we merge that.

Reply:
Changed in pushed commit.